src/DIRAC/tests/Utilities/testJobDefinitions.py

```diff
@@ -260,8 +291,11 @@ def jobWithOutputs():
     try:
         inp2 = [find_all("exe-script.py", rootPath, "DIRAC/tests/Workflow")[0]]
         J.setInputSandbox(inp1 + inp2)
-    except IndexError:  # we are in Jenkins
-        inp2 = [find_all("exe-script.py", os.environ["WORKSPACE"], "DIRAC/tests/Workflow")[0]]
+    except IndexError:
+        try:
+            inp2 = [find_all("exe-script.py", ".", "DIRAC/tests/Workflow")[0]]
```

Comment:
Can you re-write this to use something like `Path(DIRAC.__file__).parent / "tests" / "Workflow"` rather than using `find_all`?

Reply:
What's the problem with `find_all`?

Reply:
I find it's behavour confusing and needlessly complicated. Not using it would reduce 9 lines down to 2 and also be much simplier to debug if there are ever issues with it.